### ISSUE TITLE
Adding AttackVectors-Domain-List

### ISSUE THREAD

**Issue by nefty1029** [WaLLy3K/wally3k.github.io#103]
I would like to suggest this list:
https://github.com/Meldon6/AttackVectors-Domain-List

It is based on the ublock filter list here: 
https://github.com/MassMove/AttackVectors/blob/master/LocalJournals/sites-ublock-origin-filter.md

This list is designed to block phishing sites and other fake local journals domains.
These domains are also blocked by third party security extensions, such as Bitdefender Trafficlight but for those that do not want to install these extensions, this list can be used instead.

**Comment by WaLLy3K** [comment]
Just to confirm, this would be otherwise known as a "fake news" list - is that correct?

**Comment by nefty1029** [comment]
From what I have gathered from the site the list is based on, I would say yes:
https://github.com/MassMove/AttackVectors

**Comment by nefty1029** [comment]
Though some of the sites in the list are blocked by security extensions such as Bitdefender Trafficlight, so there maybe phishing or fraud involve.

**Comment by WaLLy3K** [comment]
Under my existing categories, I could potentially put it under the "other" category. However, I need to consider whether I'm willing to add this to the tick list and would love to hear some input from others whose input I value, such as @beerisgood and @XhmikosR :)

**Comment by beerisgood** [comment]
Sounds good to me

**Comment by XhmikosR** [comment]
I'd start with adding it unchecked first, and revisit in a few months maybe? I haven't used the list so I cannot comment more 🙂

**Comment by ACMEd** [comment]
Consider gorhill's response when asked (by the easylist authors, no less) to include a "fake news" blocklist with uBlock Origin:

> There are the obvious nonsense sites, but there is subjectivity involved in adding or _not_ adding some sites to such list<sup>[1]</sup>. There are always been "fake news" (somehow it's a trendy expression now as if it's a new thing). My view is that the cure is critical thinking, not a ready-to-use list in uBO. Whoever still want to use it can just import it as any other list.
> 
> [1] Example, why is The Guardian not there? [Here is a citation](https://theintercept.com/2016/12/29/the-guardians-summary-of-julian-assanges-interview-went-viral-and-was-completely-false/).

https://github.com/gorhill/uBlock/issues/2286#issuecomment-270904095

**Comment by ACMEd** [comment]
The scope of the AttackVectors list is clear: to block websites that look like local journals that were created as a part of Trump's 2020 campaign. This raises some obvious questions, such as: why aren't we blocking fake local websites that _oppose_ Trump? Either way, we should be promoting **critical thinking**, not a partisan blocklist.

**Comment by WaLLy3K** [comment]
There's been some good input here, thank you *very* much @beerisgood @XhmikosR and @ACMEd.

My biggest reservation has definitely been the politically charged nature, and having read this quote, I definitely agree with how gorhill summed it up:

> My view is that the cure is critical thinking, not a ready-to-use list in uBO. Whoever still want to use it can just import it as any other list.

I feel that people rely on my lists to protect them from a broad amount of Advertising, Tracking & Telemetry and Malicious content - and I'm confident that they do a good job at that. I can't say the same about "Misinformation" yet, and would need more than just under 2000 domains before I could consider promoting this and be even be vaguely confident that it may be effective.

TL;DR: I *would* add a "Misinformation" non-ticked category for users to consider adding if I were reasonably confident it would actually tackle that category. Currently however, I feel that this isn't enough and I don't want to commit to this yet.
